exporters/zipkin/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include_directories(include)
+find_package(CURL)
+if(CURL_FOUND)
+  add_compile_definitions(WITH_CURL)
```

Comment:
Seems `add_compile_definitions` was added to CMake since 3.12 but we support CMake 3.1 above. We may add `-DWITH_CURL` as compiler flag manually for older CMake.

Reply:
Thank.  using `add_definitions` now.